csstools/index.html

```diff
@@ -1,786 +1,389 @@
-     <li>CSS reset stylesheets are used to normalize the default CSS across browsers.</li>
-     </ul>
-</p>
-<p><a href="http://meyerweb.com/eric/tools/css/reset/">Reset.css</a> removes every default style.
-</p>
-<p><a href="http://necolas.github.com/normalize.css/">Normalize.css</a> normalizes the default styles across browsers
-(and is the preferred approach).
-</p>
-     </section>
-    </div>
-
-
-    <div class="slide normal">
-     <header><h1>CSS Grids</h1></header>
-     <section class="content">
-     	<p>A grid system lets you layout pages according to fractions of a grid (usually 12, 16, or 24 columns).</p>
-
-		<p>Example: Foundation Grid System</p>
-		<p><img src="img/foundation-grid.png" width="100%"></p>
-
-     </section>
-    </div>
-
-
-    <div class="slide normal">
-     <header><h1>CSS Grids</h1></header>
-     <section class="content">
-     	<p>A grid system lets you layout pages according to fractions of a grid (usually 12, 16, or 24 columns).</p>
-
-		<p>Example: Bootstrap Grid System</p>
-		<p><img src="img/bootstrap-grid-system.jpg" width="100%"></p>
-		
-     </section>
-    </div>
-
-
-
-    <div class="slide normal">
-     <header><h1>CSS Grids</h1></header>
-     <section class="content">
-     <p>Example: Foundation grid with mixed classes</p>
-<p>Using <a href="http://foundation.zurb.com/sites/docs/v/5.5.3/components/grid.html">Foundation CSS Grids</a>:</p>
-<pre>
-    &lt;link rel=&quot;stylesheet&quot; type=&quot;text/css&quot; href=&quot;https://cdnjs.cloudflare.com/ajax/libs/foundation-essential/6.0.6/css/foundation.css&quot;&gt;
-    &lt;div class=&quot;row&quot;&gt;
-        &lt;div class=&quot;small-2 large-4 columns&quot;&gt; ... &lt;/div&gt;
-        &lt;div class=&quot;small-4 large-4 columns&quot;&gt; ... &lt;/div&gt;
-        &lt;div class=&quot;small-6 large-4 columns&quot;&gt; ... &lt;/div&gt;
-   &lt;/div&gt;
-</pre>
-		<p><img src="img/foundation-example.png" width="100%"></p>
-		<p>Also see: <a href="http://960.gs/">960 Grids</a>, <a href="http://1kbgrid.com/">1KB Grid</a></p>
-     </section>
-    </div>
-    
-
-    <div class="slide normal">
-     <header><h1>CSS Grids</h1></header>
-     <img src="img/devtools.png" style="width:100%">
-    </div>
-
-
-    <div class="slide normal">
-     <header><h1>CSS Frameworks</h1></header>
-     <section class="content">
-		<ul>
-			<li>Why reinvent the wheel?</li>
-			<li>Responsiveness and cross browser consistency for free.</li>
-			<li>Modular and reusable components.</li>
-			<li>Usually include a reset, a grids system, styles for typography and forms. Some include support for mobile/print.</li>
-		</ul>
-     </section>
-    </div>
-
-
+       only screen and (-o-min-device-pixel-ratio: 3/2),
+       only screen and (min--moz-device-pixel-ratio: 1.5),
+       only screen and (min-device-pixel-ratio: 1.5) {
+         #logo {
+           background-image:url(images/icon@2x.png);
+         }
+       }
```

Comment:
same thing here. the  `#logo` should be farther in and the last `}` should be flush left. see negative diff for reference